### PR TITLE
Added node selector for helm chart

### PR DESCRIPTION
**Pull request by sk1p3r009**
Issue #, if available:

Description of changes: Added nodSelector label to allow options in case cluster has window and linux nodes. Set default value to linux node.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by A-Hilaly** [comment]
Hey @sk1p3r009! Thank you for you PR! These files are generated using our [code generator](https://github.com/aws-controllers-k8s/code-generator) (`ack-generate`). Can you please submit a PR to the templates that helps in generating these two files? you'll find them here https://github.com/aws-controllers-k8s/code-generator/tree/main/templates/helm

File: helm/values.yaml [hunk 1]
```diff
@@ -16,6 +16,9 @@ deployment:
   labels: {}
   containerPort: 8080
 
+nodeSelector:
+  kubernetes.io/os: linux
+
```
Comment (RedbackThomson) [inline helm/values.yaml:21]: Please move this into `deployment`. Thank you.

**Comment by sk1p3r009** [comment]
opened PR in code-generator: https://github.com/aws-controllers-k8s/code-generator/pull/185

**Pull request closed without merging**